README.md

```diff
@@ -37,30 +37,62 @@ dumps
 └── users
     └── index
         └── 0.json
+        └── 0.request_body
+        └── 0.response_body
 ```
 
-The content of each dump file is a JSON object with attributes that contain
+The content of each `#.json` dump file is a JSON object with attributes that contain
 information about the HTTP request and response. For example:
 
 ```json
 {
   "request": {
     "method": "GET",
-    "url": "http://www.example.com/test",
-    "body": ""
+    "url": "http://www.example.com/test"
   },
   "response": {
     "status": 200,
     "statusText": "OK",
     "headers": {
       "Content-Type": "text/html; charset=utf-8",
       ...
-    },
-    "body": "<p>Hello World!</p>"
+    }
   }
 }
 ```
 
+The content of `#.request_body` and `#.response_body` are the request and
+response body contents. For example:
+
+`#.request_body` :
+
+```
+foo[bar]=baz
+```
+
+`#.response_body` :
+
+```
+<p>Hello World!</p>
+```
+
+Or, for a multipart/form-data request which contains a submitted file:  
```

Comment:
The 2nd half of this comment was missed:

> Rather than catch and resolve these manually, we can have CI do it. Please take a look at PR https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/86 to resolve this and other formatting errors.

Reply:
I used the command from that PR to check and resolve formatting errors in the README, but left the changes that were outside the scope of this PR and its diff to be resolved by that more general formatting PR, particularly for timeliness. I understand better now what you were suggesting.